cloud-chatops/tests/test_base_feature.py

```diff
@@ -0,0 +1,160 @@
+"""Tests for features.base_feature.BaseFeature"""
+
+# pylint: disable=W0212
+# Disabling this as we need to access protected methods to test them
```

Comment:
If you wrap the methods you expect to be tested this is more representative of how a deriving class will use this?